packages/xsnap/package.json

```diff
@@ -12,23 +12,23 @@
   },
   "scripts": {
     "repl": "node src/xsrepl.js",
-    "build:bundle": "rollup --config rollup.config.js",
     "build:bin": "node src/build.js",
-    "build:env": "node src/build.js --show-env > dist/build-env",
-    "build:from-env": "{ cat dist/build-env; echo node src/build.js; } | xargs env",
-    "build": "yarn build:bin && yarn build:bundle && yarn build:env",
+    "build:env": "node src/build.js --show-env > build-env",
+    "build:from-env": "{ cat build-env; echo node src/build.js; } | xargs env",
+    "build": "yarn build:bin && yarn build:env",
     "clean": "rm -rf xsnap-native/xsnap/build",
     "lint": "run-s --continue-on-error lint:*",
-    "lint:js": "eslint 'src/**/*.js' 'lib/**/*.js' 'test/**/*.js' api.js",
+    "lint:js": "eslint 'src/**/*.js' 'test/**/*.js' api.js",
     "lint:types": "tsc -p jsconfig.json",
-    "lint-fix": "eslint --fix 'src/**/*.js' 'lib/**/*.js' 'test/**/*.js' api.js",
+    "lint-fix": "eslint --fix 'src/**/*.js' 'test/**/*.js' api.js",
     "test": "ava",
     "test:c8": "c8 $C8_OPTIONS ava --config=ava-nesm.config.js",
     "test:xs": "exit 0"
   },
   "dependencies": {
     "@agoric/assert": "^0.5.1",
     "@agoric/internal": "^0.2.1",
+    "@agoric/xsnap-lockdown": "^0.13.2",
```

Comment:
Hm... why does xsnap depend on xsnap-lockdown? I see references seem to be from tests, but that would suggest devDependencies. Ah... avaXS. Yeah, that should probably be its own package... eventually...

Reply:
Yeah, exactly, I wasn't certain enough about how `avaXS` is exported to move it yet. I suspect we'll need to move it before we're done with #6596, but it'll depend upon how/if things like `xsnap-lockdown` move out to a different repo, or how/if we change the way we use `lerna version` and the yarn workspace to clean up the version-number -bumping process.